src/pytest_cov/plugin.py

```diff
@@ -226,6 +244,13 @@ def pytest_runtestloop(self, session):
                 compat_session.testsfailed += 1
 
     def pytest_terminal_summary(self, terminalreporter):
+        if self._disabled:
+            markup = {'red': True, 'bold': True}
+            msg = (
+                'WARNING: Coverage disabled by user!'
```

Comment:
I can't search for the warning line. stderr us empty and stdout does nto have it.
At this point I have no idea. I need to find another way to check for this